core/types/block.go

```diff
@@ -7,16 +7,20 @@ import (
 	"fmt"
 	"github.com/Qitmeer/qitmeer/common/hash"
 	s "github.com/Qitmeer/qitmeer/core/serialization"
+	`github.com/Qitmeer/qitmeer/core/types/pow`
 	"io"
 	"math/big"
 	"time"
 )
 
 // MaxBlockHeaderPayload is the maximum number of bytes a block header can be.
 // Version 4 bytes + ParentRoot 32 bytes + TxRoot 32 bytes + StateRoot 32 bytes
-// Difficulty 4 bytes   + Timestamp 8 bytes + Nonce 8 bytes +ExNonce 8 bytes
-// --> Total 128 bytes.
-const MaxBlockHeaderPayload = 4 + (hash.HashSize * 3) + 4 + 8 + 8 +8
+// Difficulty 4 bytes   + Timestamp 4 bytes
```

Comment:
Please make the comment clear and one by one with the calculation formula 
const MaxBlockHeaderPayload = 4 + (hash.HashSize * 3) + 4 + 4 + 4 + 1 + 1 + (42*4)
the order should follow the marshalling order. I am not sure about the pow data marshalling order.